services/api/src/dao/project.js

```diff
@@ -145,7 +145,8 @@ const addProject = sqlClient => async (cred, input) => {
         },
         ${input.branches ? ':branches' : '"true"'},
         ${input.pullrequests ? ':pullrequests' : '"true"'},
-        ${input.production_environment ? ':production_environment' : 'NULL'}
+        ${input.production_environment ? ':production_environment' : 'NULL'},
+        :auto_idle
```

Comment:
@rtprio this will assume that `auto_idle` is always set via GraphQL, which is not the case, it's for example not marked as mandatory during `ProjectInput` (https://github.com/amazeeio/lagoon/pull/310/files#diff-8f08ef36ccc2a2d39d5a3ac681238b23R128)
so we need something like:

```
${input.auto_idle ? ':auto_idle' : '1'},
```